### PR TITLE
[radiothermostat] Add next scheduled set point channels

### PR DESCRIPTION
**Pull request by mlobstein**
The PR adds one last missing feature from the old mobile app. The next scheduled set point temperature and time from the programmed heating or cooling schedule can now be displayed in their own channels.
image

Also the display name of the binding was normalized to have a space between Radio Thermostat in all places that were previously inconsistent.

Note to reviewer: The RadioThermostatScheduleJson class was renamed to RadioThermostatSchedule and the new file appears in the diff as a completely new even though the method signatures and much of the code is unchanged.

### CODE REVIEW

**Comment by jlaur** [comment]
@mlobstein - many thanks! I'm wondering if it would be useful to split this into two separate channels, one for temperature and one for datetime? This would make it possible to make better use of this data, for example for automations.

**Comment by mlobstein** [comment]
> @mlobstein - many thanks! I'm wondering if it would be useful to split this into two separate channels, one for temperature and one for datetime? This would make it possible to make better use of this data, for example for automations.

Good idea, I have updated the PR to have separate channels.

**Review by jlaur — CHANGES_REQUESTED** [review]
Only a few minor comments, otherwise LGTM.

File: bundles/org.openhab.binding.radiothermostat/README.md [hunk 1]
```diff
@@ -152,33 +154,35 @@ radiothermostat:rtherm:mytherm2 "My 2nd floor thermostat" [ hostName="mythermhos
...
+Number Therm_Pmode                "Program Mode [MAP(radiotherm.map):%s_pgm]"     { channel="radiothermostat:rtherm:mytherm1:program_mode" }
+Number:Temperature Therm_Setpt    "Set Point [%d]" <temperature>                  { channel="radiothermostat:rtherm:mytherm1:set_point" }
+Number Therm_Status               "Status [MAP(radiotherm.map):%s_stus]"          { channel="radiothermostat:rtherm:mytherm1:status" }
+Number Therm_FanStatus            "Fan Status [MAP(radiotherm.map):%s_fstus]"     { channel="radiothermostat:rtherm:mytherm1:fan_status" }
+Number Therm_Override             "Override [MAP(radiotherm.map):%s_over]"        { channel="radiothermostat:rtherm:mytherm1:override" }
+Switch Therm_Hold                 "Hold"                                          { channel="radiothermostat:rtherm:mytherm1:hold" }
+Number:Temperature Therm_NextTemp "Next Set Temp [%d %unit%]" <temperature>       { channel="radiothermostat:rtherm:mytherm1:next_temp" }
+String Therm_NextTime             "Next Set Time [%s]" <time>                     { channel="radiothermostat:rtherm:mytherm1:next_time" }
```
[the remark this answers is not in the document]
Comment (mlobstein) [reply bundles/org.openhab.binding.radiothermostat/README.md]: I changed it to the string that I used in my example image.

[the remark this answers is not in the document]
Comment (jlaur) [reply bundles/org.openhab.binding.radiothermostat/README.md]: That's an additional advantage of having two channels: Possibility to format the datetime as desired. 🙂

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**